tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Integration Test for Image-rs
+
+This integration test has 2 main sub-type tests:
+* Image decryption using [ocicrypt-rs](https://github.com/confidential-containers/ocicrypt-rs)
+* Image signature verification.
+
+## Image Decryption
+
+Implemented in `decrypt_layer.rs`.
```

Comment:
Maybe rename `decrypt_layers.rs` to `decrypt_kbs_encrypted_image.rs` or `image_decryption.rs` will be better?

Reply:
Yes, changed!